### PR TITLE
Chandler Stone / One Month Submission

### PR DESCRIPTION
**Pull request by chandler-stone**
# Pull Request Name / Title
Let us know what added/removed/changed here. Make sure that you have completed the PR Checklist below before you open the PR publicly.

# Category
- [X] One-Month Project Submission
- [ ] Bug Fix

# PR Checklist
Complete this checklist before opening your pull request
- [ ] If the PR is a bug fix, the issue is linked in the PR Content
- [X] If this PR is a One-Month submission, include if you want your site to be listed publicly on our website.
- [ ] If this PR is a work in progress, it's name starts with `WIP:`. Example: `WIP: Nathan Solomon One-Month Submission`

### CODE REVIEW

**Comment by salsajeries** [comment]
ACM Logo link needs to be changed to "ACM Chapter" logo. We aren't allowed to use the ACM logo :((

**Comment by earocorn** [comment]
See Salwa

**Review by salsajeries — CHANGES_REQUESTED** [review]
ACM Logo link needs to be changed to "ACM Chapter" logo. We aren't allowed to use the ACM logo :((

File: members/ChandlerStone/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <head>
+        <title>Chandler Stone</title>
+        <link rel = "stylesheet" href = "style.css">
+        <script src="functions.js"></script>
+    </head>
+    <body>
+        <center>
+            <img src="./assets/IMG_0786.png" alt="[picture of me]" wdith="300" height="300">
+        </center>
+        <p>Hello! I am Chandler Stone, an underagraduate studying computer engineering here at the University of Alabama in Huntsville!
```
Comment (shorewind) [inline members/ChandlerStone/index.html]: undergraduate*

**Comment by chandler-stone** [comment]
Fixed wrong logo issue.
Fixed undergraduate misspelling issue.
Logo sizes were increased so "chapter" on the ACM chapter logo is clearly visual.

**Review by salsajeries — APPROVED** [review]
Changes look good!

**Pull request merged**